packages/u/usysconf-epoch/files/epoch.sh

```diff
@@ -252,6 +341,8 @@ merge_dir () {
     fi
 }
 
+console "Optimising your Solus experience, please wait."
```

Comment:
This should probably mention an expected duration, and to check the forum if it takes longer than that. Like, "This may take several minutes, but should not take more than ten minutes. If it does please check the Solus forum for guidance"

CC @TraceyC77 @EbonJaeger  on wording.

Reply:
Is line length something we have to worry about? If not, then the wording proposed by Reilly sounds fine to me, as long as the duration used is reasonable for real-world expectations. Our Matrix channels could also be mentioned in place of, or in addition to, the forum mention.

Reply:
What do you think about showing the following after 5 minutes?

![Screenshot_solus4_2024-08-18_21:15:17](https://github.com/user-attachments/assets/ad093d46-14d4-482e-a6d7-032fca3b7b15)

Reply:
I like that. It looks good to me.